### DISCUSSION TITLE
WHY DEBRID?

### DISCUSSION THREAD

**Discussion by MeirG9** [itsToggle/plex_debrid#34] [Ideas]
First of all, amazing work!
I really liked the way you managed to make things work.
I wonder, if there is a possibility to use PLEX discovery, why use Debrid?
There is an option to write a code that will access RARBG, search for the movie selected in "Discovery" and download the torrent to the "Plex" folder.
I think it is possible, I would love to hear your opinion. Any ideas on how to implement this?

**Comment by itsToggle** [comment]
For simple Torrent downloading, the best option out there is Radarr and Sonarr. They support the Plex watchlist as input for torrent scraping.
The goal of this script is to make use of the debrid services cache. The debrid service cache allows you to watch content almost instantaniously, since you dont actually need to download the torrent. Thats why it only Takes 10-20 seconds from watchlisting something to watching it :)
Using Radarr and Sonarr, downloading a Torrent will Take a few minutes even on the best Internet connection.

**Reply by bitsper2nd** [reply]
Have you considered adding [support for Put.io](http://help.put.io/en/articles/3480094-plex-rclone)?

**Reply by itsToggle** [reply]
Ill have a look at their API :)

**Reply by itsToggle** [reply]
> Have you considered adding [support for Put.io](http://help.put.io/en/articles/3480094-plex-rclone)?

Put.io cant be integrated in the same way as the other debrid services, as there is no way to bulk-check for cached releases. I can integrate it by simply creating a new transfer for the best release that was found, regardless of its cached status. This would mean that a lot of uncached releases will be added though.
